### PR TITLE
[BotW] Add Intel iGPU workarounds

### PR DESCRIPTION
**Pull request by Alstura**
Mods for excess intel bloom and fixing flipped light reflections

### CODE REVIEW

File: Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Remove Intel Excess Bloom"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Excess Intel Bloom"
+description = Removes excess bloom common in 23.xx and Intel older drivers. Apply before shadow fixes for no-cel. -Altros
```
Comment (Crementif) [inline Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt]: First of all, it's fine to add credits to the description (I feel like it should be more third-person-ish like a normal "Credits to Altros"). It's not a personal pack that you're giving out on Discord.

Also, the description isn't grammatically correct. I would just suggest changing the whole thing to something like:
`This workaround is for people who use the older Intel integrated GPU drivers which offer better performance but trade in graphical bugs. If you use this pack, make sure you install the 23.XX drivers or lower (see this link [insert link]). This pack fixes most issues with the older driver to look like the newer drivers.`

Maybe you can make it shorter or easier for people. I also recommend including a link to the drivers, I couldn't even find them.

File: Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = Fix Flipped Intel Lights"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Flipped Intel Lights"
+description = Fixes flipped lights common in 23.xx and older Intel drivers. -Altros
```
Comment (Crementif) [inline Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt]: Basically, this should probably have the same description as the other rules.txt file.

File: Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = Fix Flipped Intel Lights"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Flipped Intel Lights"
```
Comment (Crementif) [inline Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt]: You should move this path to `The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Remove Flipped Intel Lights`.

File: Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt [hunk 4]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = Fix Flipped Intel Lights"
```
[the remark this answers is not in the document]
Comment (Crementif) [reply Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt]: I think you should also change this to "Fix Flipped Lighting On Older Intel iGPU Drivers" or something similar.

File: Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt [hunk 5]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Remove Intel Excess Bloom"
```
Comment (Crementif) [inline Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt]: Should change this to "Fix Excessive Bloom On Older Intel iGPU Drivers".

File: Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt [hunk 6]
```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Remove Intel Excess Bloom"
+path = "The Legend of Zelda: Breath of the Wild/Graphics/Intel/Remove Excess Intel Bloom"
```
Comment (Crementif) [inline Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt]: You should move this path to `The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Remove Excessive Intel Bloom`.

File: Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Alstura) [reply Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt]: Modified title as suggested. Added " back in its appropriate place at the beginning of the name line.

File: Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt (hunk 1 above)
Comment (Alstura) [reply Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt]: Moved credits to a comment for all rules.txt.  Modified path to /Workarounds/Intel/ for all packs. Modified description for "bloom" pack to.  description = This workaround targets users of Intel iGPU drivers 23.xx and older experiencing bloom issues. These drivers may offer better performance but have graphical glitches which. This pack aims to fix the graphical issues so the older drivers create visuals similar to the new drivers. These older drivers may be found on Intel's site https://downloadcenter.intel.com/download/27484/Graphics-Intel-Graphics-Driver-for-Windows-15-65-.

File: Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt (hunk 2 above)
Comment (Alstura) [reply Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt]: Moved credits and pack to Workarounds/Intel iGPU

File: Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt (hunk 3 above)
Comment (Alstura) [reply Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt]: Description updated 

description = This workaround targets users of Intel iGPU drivers 23.xx and older experiencing flipped lighting issues. These drivers may offer better performance but have graphical glitches. This pack aims to fix the graphical issues so the older drivers create visuals similar to the new drivers. These older drivers may be found on Intel's site https://downloadcenter.intel.com/download/27484/Graphics-Intel-Graphics-Driver-for-Windows-15-65-.

Comment (Alstura) [reply Mods/BreathOfTheWild_Intel Fix Flipped Lights/rules.txt]: Path updated
path = "The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Fix Flipped Intel Lights"

File: Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt (hunk 5 above)
Comment (Alstura) [reply Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt]: Name modified as suggested

File: Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt (hunk 6 above)
Comment (Alstura) [reply Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt]: Path modified to
path = "The Legend of Zelda: Breath of the Wild/Workarounds/Intel iGPU/Remove Excess Intel Bloom"

**Pull request merged**